fritz/device.go

```diff
@@ -29,3 +29,8 @@ func (d *Device) IsSwitch() bool {
 func (d *Device) IsThermostat() bool {
 	return bitMasked{Functionbitmask: d.Functionbitmask}.hasMask(64)
 }
+
+// Returns is the device has a temperature sensor
```

Comment:
Can u fix the godoc issue https://codeclimate.com/github/bpicode/fritzctl/pull/122